src/components/YieldPools/ElasticFarmGroup/index.tsx

```diff
@@ -741,7 +683,7 @@ const ProMMFarmGroup: React.FC<Props> = ({ address, onOpenModal, farms }) => {
   }
 
   return (
-    <FarmContent>
+    <FarmContent data-testid="farm-block">
```

Comment:
remove this

Reply:
no. I added this for QA. They can use that attribute to check performance and compare with production